src/gui/kbperf.cpp

```diff
@@ -133,6 +136,10 @@ void KbPerf::load(CkbSettingsBase& settings){
         }
     }
     SGroup group(settings, "Performance");
+    {
```

Comment:
There's no need to make a new scope here

Reply:
I followed the other examples and it hides the local `s_muteDev`, that is only there for readability.

Reply:
If you are referring to other cases such as https://github.com/ckb-next/ckb-next/blob/8348873a1ad6ab9c471b738c188887c648073e5a/src/gui/kbperf.cpp#L239-L251

then it's used to explicitly call the SGroup destructor to go one level up in the QSettings group hierarchy.
https://github.com/ckb-next/ckb-next/blob/2dbf7221c416a7648981c1312b8ccc3f1a5ab48e/src/gui/ckbsettings.h#L99

Reply:
I see. I removed the extra scope.